### PR TITLE
Removed cloudant and logging. Also added test to scripts

### PR DESCRIPTION
**Pull request by stevenpkg**
Removed conversation logging as not required now

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,17 +23,16 @@
     "vcap_services": "^0.2.0",
     "watson-developer-cloud": "^1.12.2",
     "watson-speech": "^0.19.1",
-    "cloudant": "1.4.1",
     "express-csv": "0.6.0",
-    "nano" : "*",
-    "uuid": "2.0.2"
+    "nano" : "*"
```
Comment (germanattanasio) [inline package.json]: nano is not required

Comment (germanattanasio) [reply package.json]: And if it is, never use *

File: test/test.express.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * limitations under the License.
+ */
+
+'use strict';
+
+var app = require('../app');
+var bodyParser = require('body-parser');
+var request = require('supertest');
+
+app.use(bodyParser.json());
```
Comment (germanattanasio) [inline test/test.express.js]: `app.js` already adds the json parser

**Review by germanattanasio — CHANGES_REQUESTED** [review]

**Comment by stevenpkg** [comment]
Have made the changes you requested

**Comment by germanattanasio** [comment]
@stevenpkg the changes look good. I still think there are unused dependencies like `express-csv`
If you install `depcheck` with: 
```
npm install depcheck -g
```

Then you can run
```
depcheck
```

And see the unused dependencies.

**Comment by stevenpkg** [comment]
Thanks for letting me know about depcheck. Cleaned up the following dependencies:
Unused dependencies
* chai
* request
* watson-speech
Unused devDependencies
* gulp-fontmin
* gulp-if
* gulp-imagemin
* gulp-postcss
* gulp-preprocess
* gulp-uglify
* nock
* uglifyjs
* whatwg-fetch
Missing dependencies
* eve

**Comment by stevenpkg** [comment]
I see mocha is in both dependencies, is this correct?

},
  "author": "IBM Corp.",
  "dependencies": {
    "body-parser": "1.15.1",
    "compression": "1.6.2",
    "dotenv": "^2.0.0",
    "express": "4.13.4",
    "mocha": "^2.5.3",
    "vcap_services": "^0.2.0",
    "watson-developer-cloud": "^1.12.2",
    "express-csv": "0.6.0"
  },
  "devDependencies": {
    "babel-eslint": "^6.0.4",
    "eslint": "^2.8.0",
    "eslint-config-airbnb-es5": "^1.0.9",
    "eslint-plugin-react": "^5.1.1",
    "gulp": "^3.9.1",
    "istanbul": "^0.4.2",
    "gulp-clean-css": "2.0.7",
    "gulp-develop-server": "^0.5.1",
    "gulp-load-plugins": "^0.7.1",
    "gulp-size": "^1.1.0",
    "gulp-sourcemaps": "^1.6.0",
    "gulp-useref": "^3.0.8",
    "mocha": "^2.4.5",
    "supertest": "^1.2.0",
    "eve": "^0.3.1"
  }

**Comment by AthifAlpha01** [comment]
Deployed but there is an error saying " The service may be down at the moment; please check here for the current status.
If the service is OK, the app may not be configured correctly, please check workspace id and credentials for typos.
If the service is running and the app is configured correctly, try refreshing the page and/or trying a different request."
How can I overcome this ?

**Pull request merged**